ceph_salt/validate/config.py

```diff
@@ -66,6 +96,19 @@ def validate_config(host_ls):
             return 'No external time servers specified in config'
         if not time_server_is_minion and external_time_servers:
             return not_minion_err.format('external time servers')
+        # assert that external_time_servers are sane
+        if external_time_servers:
+            external_time_server_ip_addresses = []
+            ahost = None
+            for ahost in external_time_servers:
+                ipaddrs = host_resolvable_to_ip(ahost)
```

Comment:
I'm afraid this is more complex than anticipated.

This PR is checking if `salt-master` (the host where `ceph-salt` is running) can resolve `external time servers`, but the host that should be able to resolve it is the `time_server_minion`, not the master.

I wonder if salt `network` module can help here https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.network.html

Some examples:

```
salt '*' network.ping pool.ntp.org return_boolean=True

salt '*' network.connect pool.ntp.org port=123 proto=udp
```

Example of how to execute a salt module on a minion (from `ceph-salt` code):
```
returns = SaltClient.local().cmd(time_server_host, 'network.connect', [ahost, 'port=123', 'proto=udp'])
```

But for some reason, `salt '*' network.connect pool.ntp.org port=123 proto=udp` is always failing on my tests, so I wonder if the best we can to is simply check if `ping` works?